src/article/editor/EditorPackage.js

```diff
@@ -213,10 +234,12 @@ export default {
     config.addLabel('insert-table', 'Table')
     config.addIcon('insert-table', { 'fontawesome': 'fa-table' })
 
+    config.addTool('edit-block-formula', EditDispFormulaTool)
```

Comment:
We should use a consistent name, either `disp-formula` or `block-formula`
@michael pls your preference

Reply:
My personal preference is actually 'block-formula' similar to 'block-quote'

Reply:
In the AMS extension for Lens we used 'equation' here.